### PR TITLE
ocw studio config

### LINKED ISSUE #16: create ocw-studio.yml file
This repo should contain an `ocw-studio.yml` file that describes the types of pages that can be created and their fields, somewhat like a Netlify CMS configuration:

image
https://www.netlifycms.org/docs/configuration-options/

### PR DESCRIPTION
**Pull request by gumaerc**
#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-course-hugo-starter/issues/16

#### What's this PR do?
This PR adds a rough draft of what the `ocw-studio` configuration should be for this starter repo.

#### How should this be manually tested?
There's no way this can be tested at the moment, this is just a first draft of what the configuration should look like

### CODE REVIEW

File: ocw-studio.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+metadata:
```
Comment (Ferdi) [inline ocw-studio.yml]: Please remove metadata from this version

File: ocw-studio.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  - label: "Term"
+    name: "term"
+    widget: "string"
+    default: ""
+  - label: "Level"
+    name: "level"
+    widget: "string"
+    default: ""
+collections:
+  - name: "pages" # Used in routes, ie.: /admin/collections/:slug/edit
```
Comment (Ferdi) [inline ocw-studio.yml]: Please Add 2 collections , "Page" and "Resource", each with minimal fields for now

File: ocw-studio.yml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    label: "Pages" # Used in the UI
+    label_singular: "Page"
+    folder: "sections" # The folder under the content folder these items should be placed in
```
Comment (Ferdi) [inline ocw-studio.yml]: Please make sure this follows exactly the netlifycms schema for now.

**Review by Ferdi — CHANGES_REQUESTED** [review]

File: ocw-studio.yml (hunk 3 above)
Comment (gumaerc) [reply ocw-studio.yml]: Which property were you referring to?  Aside from `metadata` , I didn't add anything that isn't used in a Netlify CMS config following this example:

https://github.com/netlify/netlify-cms/blob/master/dev-test/config.yml

File: ocw-studio.yml (hunk 2 above)
Comment (gumaerc) [reply ocw-studio.yml]: I can do this, but I posed this question to Gavin and I'm wondering what you think: isn't any type of site we create going to need a place to upload resources?  I thought we had discussed the fact that people will upload static resources like PDFs and images, but the Resources section here does not represent actual pages.  For example, you would go into Resources to upload a PDF, but a PDF Viewer page would still need to be created.  In Netlify, collections that you define directly represent pages that can be created.  If you were uploading an image into resources, I don't imagine you would want to create a page representing just that image.

**Comment by gumaerc** [comment]
After talking with Ferdi on Zoom I simplified this.  I was under the impression that this file needed to define more than it does.  It should be 1:1 with what the UI shows.

**Comment by gumaerc** [comment]
@Ferdi This should be all set now.  I left off resource type, authors and license for now.  I think these will be dropdowns with specific choices right?

**Review by noisecapella — APPROVED** [review]

**Pull request merged**